examples/httpd/convox.yml

```diff
@@ -2,3 +2,21 @@ services:
   web:
     image: httpd
     port: 80
+  worker:
+    build: ./worker
+
+timers:
+  cleanup:
+    command: ./cmd.sh
+    schedule: "* * * * *"
+    service: worker
```

Comment:
Rename the timer to a more descriptive (e.g. `example`) and link it to the `web` service.

Reply:
hey, @Twsouza. What do you mean about "link it to the web service"?

Reply:
instead of using the `worker` on `services` for the timer, use the `web` service:

```
example:
    command: ./cmd.sh
    schedule: "* * * * *"
    service: worker
```

You may need to create a Dockerfile to the `web` service to copy the shell script.

Reply:
hey @Twsouza, thanks for your feedback, man. I've just updated and remove the dependency from the concurrency task. Before merge it, I'll do the respective tests.